src/plugins/gui/providers/bityProvider.ts

```diff
@@ -609,10 +610,18 @@ export const bityProvider: FiatProviderFactory = {
                   return
                 }
 
-                if (isBuy) {
-                  await completeBuyOrder(approveQuoteRes, showUi)
-                } else {
-                  await completeSellOrder(approveQuoteRes, coreWallet, showUi, fiatCurrencyCode, tokenId)
+                try {
+                  if (isBuy) {
+                    await completeBuyOrder(approveQuoteRes, showUi)
+                  } else {
+                    await completeSellOrder(approveQuoteRes, coreWallet, showUi, fiatCurrencyCode, tokenId)
+                  }
+                } catch (e: any) {
+                  if (e.message === SendErrorBackPressed) {
+                    // Do nothing
+                  } else {
+                    throw e
+                  }
```

Comment:
It is never safe to read `e.message`, until you have determined that this thing is actually an error. Do this instead:

```ts
catch (e: unknown) {
  if (e instanceof Error && e.message === SendErrorBackPressed) {
    return
  }
  throw e
}
```